state/dbStateManager.go

```diff
@@ -844,7 +844,7 @@ func (list *DBStateList) FixupLinks(p *DBState, d *DBState) (progress bool) {
 				continue
 			}
 			amt := primitives.CalculateCoinbasePayout(ia.Efficiency)
-			if amt == 0 {
+			if amt == 0 || amt > constants.COINBASE_PAYOUT_SERVER_LIMIT {
```

Comment:
When the fed server payout that occurs every 25 blocks exceeds the limit, don't load it onto the admin block.

Reply:
That `amt` is coming from calculating the coinbase payout on a per authority basis. Not the sum of all the payouts.

Your comment on `COINBASE_PAYOUT_SERVER_LIMIT` says that it is the payout limit of 100 fed nodes, and this conditional is checking the 100 fed node limit against a single node payout. 

Either you'd want to change the conditional to be checking the limit of a single node, or summation all the payouts, and then do a check.